src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.html

```diff
@@ -29,7 +29,13 @@
           </div>
           <div class="col-sm-6 col-lg-6 metadata"
                *ngIf="metadata">
-            <legend>{{ metadataTitle }}</legend>
+            <legend>{{ metadataTitle }}
+              <button type="button"
```

Comment:
Can we add a `Reload` tooltip here?

Reply:
Yup. But currently I am looking at some implementation where the refresh happens automatically.